scripts/pi-setup/auto-install.sh

```diff
@@ -19,16 +19,16 @@ SCRIPT_DIR="$ROOT_DIR/scripts"
 PI_SCRIPT_DIR="$SCRIPT_DIR/pi-setup"
 UBUNTU_SCRIPT_DIR="$SCRIPT_DIR/ubuntu"
 
-printf "\n\n************************************************************"
-printf "\n*** Installing Raspberry Pi Dependencies ***\n"
+source "$SCRIPT_DIR/utils.sh"
+
+print_start_of_script
+
+print_instalation_step "Installing Raspberry Pi Dependencies"
 $PI_SCRIPT_DIR/install-pi-dependencies.sh
-if [ $? -ne 0 ]; then
-    echo "### Exit with Error ###"
-    exit 1
-fi
+verify_return_code
 
+print_instalation_step "Installing Ubuntu Dependencies"
```

Comment:
Doesn't the auto-install script do more than just installing dependencies?

Reply:
Yes, but I couldn't think of a better name. Would you like to suggest something?

Reply:
Run ubuntu auto-install?